src/main/scala/cromwell/engine/backend/package.scala

```diff
@@ -3,6 +3,11 @@ package cromwell.engine
 import scala.concurrent.{ExecutionContext, Future}
 
 package object backend {
+  // Ordered by shards, and then ordered by attempts
+  type AttemptedCallLogs = Seq[Seq[CallLogs]]
```

Comment:
The more I look at this stuff, the more wtf... Why do `scala.collection.immutable.(Seq|Traversable|Iterable).newBuilder` each return a `mutable.ListBuffer`?